src/canary_cmake/__init__.py

```diff
@@ -122,3 +163,20 @@ def canary_resource_pool_fill(config: canary.Config, pool: dict[str, dict[str, A
         pool["resources"].update(resource_specs["local"])
         if "cpus" not in pool["resources"]:
             pool["resources"]["cpus"] = cpu_spec
+
+
+@canary.hookimpl(wrapper=True)
+def canary_cdash_artifacts(case: canary.TestCase) -> Generator[None, None, list[dict[str, str]]]:
+    """Default implementation: return the test case's keywords"""
```

Comment:
```suggestion
    """Default implementation: return the test case's artifact specifications"""
```
Should `canary.TestCase` use this Schema to validate the artifacts specification?